Makefile

```diff
@@ -27,14 +27,13 @@ sani-vm: $(vm_tests:=.format) $(vm_fillers:=.format) \
          $(vm_tests:=.filled)
 
 # TODO: enable $(gs_fillers:=.valid) $(gs_tests:=.format) $(gs_fillers:=.format)
-sani-gs: $(gs_tests:=.valid) \
```

Comment:
can you make it a separate pr and commit as a final version of this file to develop